### PR TITLE
Added BottomTabNavigator

### LINKED ISSUE #9: Add Bottom Tab Navigator
**Is your feature request related to a problem? Please describe.**
Bottom Tab Navigator helps in navigation between screens.

**Describe the solution you'd like**
Implement Bottom Tab Navigator for navigation purposes

### PR DESCRIPTION
**Pull request by AbhiKul10**
**Describe the Feature you Added**
Added Bottom Tab Navigator, so by which User may Navigate between screens.

**How will your Feature Help?**
Bottom Tab Navigator will let the user navigate between Screens.

Fixes #9 

<p align="center"  float="left">

</p>

### CODE REVIEW

**Comment by AbhiKul10** [comment]
@shehand Please Review my PR!

File: src/screens/ProfileScreen/styles.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+    LoginButtonView : {
+        width: "100%", 
+        height: 50, 
+        backgroundColor: "#51AD28",
```
Comment (shehand) [inline src/screens/ProfileScreen/styles.js:7]: If these color codes are global, which means repeated in different files, put them into a single file and import them. In this way app theme can be easily modify.

File: src/screens/StartScreen/StartScreen.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import LoginSignupScreen from '../LoginSignupScreen/LoginSignupScreen';
 import { Spinner, Button } from '../../components/common';
 import { firebase } from '../../config/config';
 import HomeScreen from '../HomeScreen/HomeScreen';
```
Comment (shehand) [inline src/screens/StartScreen/StartScreen.js:7]: If you see any unused imports, please remove them.

**Review by shehand — CHANGES_REQUESTED** [review]

File: src/screens/ProfileScreen/styles.js (hunk 1 above)
Comment (AbhiKul10) [reply src/screens/ProfileScreen/styles.js]: Done!

File: src/screens/StartScreen/StartScreen.js (hunk 2 above)
Comment (AbhiKul10) [reply src/screens/StartScreen/StartScreen.js]: Okay!

File: src/components/LoginForm/styles.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { StyleSheet } from 'react-native';
+import {COLOR_PRIMARY, COLOR_GRAY, COLOR_GRAY1 } from '../../config/styles';
```
Comment (shehand) [inline src/components/LoginForm/styles.js:2]: suggestion for COLOR_GRAY and COLOR_GRAY1 to change COLOR_GRAY_PRIMARY and COLOR_GRAY_SECONDARY. Be specific when you declaring global variables like these. It is fine if they are method local or class local. But not a good practice when comes to things like these

File: src/config/styles.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// Theme Colors 
+export const COLOR_PRIMARY = '#51AD28';
+export const COLOR_GRAY = '#dbdbdb'; 
+export const COLOR_GRAY1 = '#d4d4d4'; 
+export const COLOR_LIGHTP = '#9cd683';
```
Comment (shehand) [inline src/config/styles.js:5]: Is it meant to be LIGHTP?

**Review by shehand — CHANGES_REQUESTED** [review]

File: src/components/LoginForm/styles.js (hunk 3 above)
Comment (AbhiKul10) [reply src/components/LoginForm/styles.js]: Sure! I'll remember that!

File: src/config/styles.js (hunk 4 above)
Comment (AbhiKul10) [reply src/config/styles.js]: Changed name to COLOR_LIGHT_PRIMARY

**Review by shehand — APPROVED** [review]

**Pull request merged**